moveit_planners/trajopt/trajopt/CMakeLists.txt

```
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(trajopt)
+
+add_compile_options(-std=c++11 -Wall -Wextra)
+
+find_package(catkin REQUIRED COMPONENTS
+  trajopt_sco
+  trajopt_utils
+  roscpp
+)
+
+find_package(Eigen3 REQUIRED)
+find_package(Boost COMPONENTS system python thread program_options REQUIRED)
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(JSONCPP jsoncpp)
+
+catkin_package(
+  INCLUDE_DIRS
+    include
+  LIBRARIES
+    ${PROJECT_NAME}
+  CATKIN_DEPENDS
+    trajopt_sco
+    trajopt_utils
+    roscpp
```

```suggestion
  CATKIN_DEPENDS
    moveit_core
    trajopt_sco
    trajopt_utils
    roscpp
```

Adding `moveit_core` here as well.